lib/rspec/collection_matchers/have.rb

```diff
@@ -24,6 +24,12 @@ def relativities
         }
       end
 
+      if RUBY_VERSION == '1.9.2'
+        def to_ary
+          [self]
+        end
+      end
```

Comment:
Because of the "recent" changes for multiple diffs in `rspec-expectations` from compound matchers theres an array that gets flattened, and on `1.9.2` `to_ary` returns `self` by default and fails. Its a weird bug thats taken me a while to debug (and causes a further bug with bins stubs FML).